txdbus/authentication.py

```diff
@@ -141,7 +146,7 @@ def _auth_DATA(self, line):
                 self.sendAuthMessage(b'ERROR ' + str(e).encode())
```

Comment:
This may fail for non-ascii error messages. It may be safer to use `.encode("unicode-escape")` instead.

(Again the spec mandates these strings to be ASCII.)